apps/main-landing/src/components/hero-section/components/collaborations-showcase.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  AavegotchiLogo,
+  AcrossLogo,
+  AgoraFullLogo,
+  GitcoinLogo,
+  ParallelLogo,
+  PolymarketLogo,
+  SnapshotFullLogo,
+  TallyLogo,
+} from '@idriss-xyz/ui/branding-assets';
+import { Button } from '@idriss-xyz/ui/button';
+import { Marquee } from '@idriss-xyz/ui/marquee';
+import { classes } from '@idriss-xyz/ui/utils';
+
+type CollaborationsShowcaseProperties = {
+  className?: string;
+};
+
+const logoClassName =
+  'mx-2.5 h-5.5 w-fit text-neutralGreen-500 lg:h-10 hover:text-neutralGreen-700';
+
+const collaborationLinks = [
+  {
+    logo: <ParallelLogo className={logoClassName} />,
+    link: 'https://x.com/IDriss_xyz/status/1729258820441170160',
+  },
+  {
+    logo: <AcrossLogo className={logoClassName} />,
+    link: 'https://x.com/AcrossProtocol/status/1782919859439092151',
+  },
+  {
+    logo: <PolymarketLogo className={logoClassName} />,
+    link: 'https://x.com/Polymarket/status/1778470611083313375',
+  },
+  {
+    logo: <GitcoinLogo className={logoClassName} />,
+    link: 'https://x.com/gitcoin/status/1786121984105144501',
+  },
+  {
+    logo: (
+      <AavegotchiLogo
+        className={classes(logoClassName, 'h-[17.42px] lg:h-[31.7px]')}
+      />
+    ),
+    link: 'https://x.com/IDriss_xyz/status/1750851248121930167',
+  },
+  {
+    logo: <TallyLogo className={logoClassName} />,
+    link: 'https://x.com/IDriss_xyz/status/1807798872930202007',
+  },
+  {
+    logo: (
+      <AgoraFullLogo
+        className={classes(logoClassName, 'h-[17.42px] lg:h-[31.7px]')}
```

Comment:
does it work to have values after commas in px values? :D maybe it should be 18/32 or 17/32?

Reply:
Yeah, it's working. Wdyt about keeping px for simplicity?
![image](https://github.com/user-attachments/assets/64f32227-4766-4ecf-9c3c-1c16ff4cd1fa)